### DISCUSSION TITLE
Mo-coefficients  in dhf is changing everytime. It is effecting my ao to mo transformation. how to make Mo-coefficients stable.

### DISCUSSION THREAD

**Discussion by ironhide210** [pyscf/pyscf#2646] [Q&A] [answered]

**Comment by tberkel** [comment, marked as answer]
This is unavoidable if there are degeneracies in the MO energies. See [here](https://github.com/pyscf/pyscf/issues/1717) for example.

**Comment by ironhide210** [comment]
how to avoid it  using molecular symmetry.

**Comment by ironhide210** [comment]
it doesnt change for RHF and X2C spinfree case
